### PR TITLE
myc-27 ui previews

### PR DESCRIPTION
**Pull request by Faisal25DEC**

### CODE REVIEW

File: components/SplitsTypes/SubmitForm.tsx [hunk 1]
```diff
@@ -1,49 +1,44 @@
...
-          label="Both"
-        />
+        {splitTypes.map((splitTypeInfo, index) => (
+          <CheckboxLabel
+            key={index}
+            label={splitTypeInfo.label}
+            checked={splitType === splitTypeInfo.type}
+            onClick={() => setSplitType(splitTypeInfo.type)}
+          />
+        ))}
```
Comment (sweetmantech) [inline components/SplitsTypes/SubmitForm.tsx:29]: 🧠 👏 
- less is more
- simple is best
- small is beautiful

File: components/SplitsTypes/constants.ts
Comment (sweetmantech) [inline components/SplitsTypes/constants.ts:1]: could you please move this to the `libs/` directory?
architecture
- `/components` - only react components
- `/lib` - `functions`, `types` and other `utility functions`.

File: components/Vote/Descriptions.tsx [hunk 2]
```diff
@@ -11,8 +14,10 @@ const Descriptions = () => {
       <p className="font-roboto text-2xl">
         None of the parties may perform legally relevant acts on the musical
         work without the written authorization of the{' '}
-        <span className="text-danger-dark font-rubik">51%</span> of the
-        ownership, such as but not limited to the following:
+        <span className="text-danger-dark font-rubik">
+          {votePercentage || '51'}%
```
Comment (sweetmantech) [inline components/Vote/Descriptions.tsx]: What do you think about having the default value of `votePercentage` be `51%` so we can remove this fallback? Also provides accurate default value for votePercentage across the `provider` usage right???

Comment (Faisal25DEC) [reply components/Vote/Descriptions.tsx]: Sure

File: components/SplitsTypes/constants.ts
Comment (Faisal25DEC) [reply components/SplitsTypes/constants.ts:1]: Ok

File: components/SplitsTypes/SubmitForm.tsx (hunk 1 above)
Comment (Faisal25DEC) [reply components/SplitsTypes/SubmitForm.tsx:29]: Thanks

**Review by sweetmantech — APPROVED** [review]

**Pull request merged**